client/src/pages/SplashPage/SplashPage.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import DiamondActiveSrc from "../../assets/diamondActive.svg";
+import { Diamond } from "../../components/WeekListItem/WeekListItem";
+import styled from "styled-components/macro";
+
+const CenterLogo = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+`;
+
+const AnimatedDiamond = styled(Diamond)`
+background-image: url(${DiamondActiveSrc});
+  height: 10rem;
+  width: 10rem;
+  padding-right: 5px;
+  padding-bottom: 15px;
```

Comment:
You can condense these lines into one. 
```suggestion
  padding: 0 5px 15px 0px;
```